nightingale_dispatcher/src/nightingale_dispatcher/navigate_task.py

```diff
@@ -1,18 +1,30 @@
 #!/usr/bin/env python3
 
 import rospy
-
 from nightingale_dispatcher.task import Task
+import actionlib
+from actionlib_msgs.msg import GoalStatus
+from nightingale_msgs.msg import RoomRunnerAction, RoomRunnerActionGoal
 
 
 class NavigateTask(Task):
-    # Go to position with given context
-    # TODO define parameters
-    def __init__(self, *args, **kwargs):
-        raise NotImplementedError()
+    def __init__(self):
+        action_name = "/nightingale_navigation/room_runner"
```

Comment:
Nit: Can we move this to a static constant member of NavigateTask

Reply:
i didnt make it a member bc its not used anywhere aside from init